python/proton/_wrapper.py

```diff
@@ -78,11 +83,12 @@ def __init__(self, impl_or_constructor, get_context=None):
 
         if get_context:
             record = get_context(impl)
-            attrs = pn_void2py(pn_record_get(record, PYCTX))
-            if attrs is None:
+            attrs = pn_record_get(record, PYCTX)
+            if attrs is None or attrs == ffi.NULL:
                 attrs = {}
+                attrs = ffi.new_handle({})
                 pn_record_def(record, PYCTX, PN_PYREF)
```

Comment:
@jiridanek  The error is cause by this line. After executing `pn_record_def` and `pn_record_set` it crashes without any useful error message. Remaining test cases also depend on the _wrapper.py